electrum/plugins/bitbox02/bitbox02/bitbox02/bitbox02.py

```diff
@@ -0,0 +1,492 @@
+        try:
+            response = self._msg_query(request, expected_response="check_backup")
+        except Bitbox02Exception as err:
+            if err.code == ERR_GENERIC:
+                return None
+            raise
+        return response.check_backup.id
+
+    def show_mnemonic(self) -> bool:
+        """
+        Returns True if mnemonic was successfully shown and confirmed.
+        Returns False otherwise.
+        """
+        # pylint: disable=no-member
+        request = hww.Request()
+        request.show_mnemonic.CopyFrom(mnemonic.ShowMnemonicRequest())
+        try:
+            self._msg_query(request, expected_response="success")
+        except Bitbox02Exception as err:
+            if err.code == ERR_GENERIC:
+                return False
+            raise
+        return True
+
+    def btc_pub(
+        self,
+        keypath: List[int],
+        coin: btc.BTCCoin = btc.BTC,
+        output_type: btc.BTCPubRequest.OutputType = btc.BTCPubRequest.XPUB,
+        script_type: btc.BTCScriptType = btc.SCRIPT_UNKNOWN,
+        display: bool = True,
+    ) -> str:
+        """
+        keypath is a list of child derivation numbers.
+        e.g. m/44'/0'/1'/5 corresponds to [44+HARDENED, 0+HARDENED, 1+HARDENED, 5].
+        """
+        # pylint: disable=no-member,too-many-arguments
+        request = hww.Request()
+        request.btc_pub.CopyFrom(
+            btc.BTCPubRequest(
+                coin=coin,
+                keypath=keypath,
+                output_type=output_type,
+                script_type=script_type,
+                display=display,
+            )
+        )
+        return self._msg_query(request).pub.pub
+
+    # pylint: disable=too-many-arguments
+    def btc_sign(
+        self,
+        coin: btc.BTCCoin,
+        script_type: btc.BTCScriptType,
+        bip44_account: int,
+        inputs: List[BTCInputType],
+        outputs: List[BTCOutputType],
+        version: int = 1,
+        locktime: int = 0,
+    ) -> List[Tuple[int, bytes]]:
+        """
+        coin: the first element of all provided keypaths must match the coin:
+        - BTC: 0 + HARDENED
+        - Testnets: 1 + HARDENED
+        - LTC: 2 + HARDENED
+        script_type: type of all inputs and change outputs. The first element of all provided
+        keypaths must match this type:
+        - SCRIPT_P2PKH: 44 + HARDENED
+        - SCRIPT_P2WPKH_P2SH: 49 + HARDENED
+        - SCRIPT_P2WPKH: 84 + HARDENED
+        bip44_account: Starting at (0 + HARDENED), must be the third element of all provided
+        keypaths.
+        inputs: transaction inputs.
+        outputs: transaction outputs. Can be an external output
+        (BTCOutputExternal) or an internal output for change (BTCOutputInternal).
+        version, locktime: reserved for future use.
+        Returns: list of (input index, signature) tuples.
+        Raises Bitbox02Exception with ERR_USER_ABORT on user abort.
+        """
+        import pickle
```

Comment:
remove

Reply:
Will be removed once I copy over the latest version of the library ;)